tests/profiling/settings.js

```diff
@@ -1,22 +1,25 @@
 const {getAssets} = require('./dao.js');
+//const redisStore = require('ha-store-redis');
 
 module.exports = {
     test: {
         sampleFile: './sample.txt',
     },
     setup: {
         resolver: getAssets,
+        // store: redisStore('footage-api-ha-cache', { host: '0.0.0.0', port: 6379 }),
         uniqueParams: ['language'],
         cache: { limit: 5000, ttl: 300000 },
         batch: { tick: 10, max: 50 },
         retry: { base: 1, step: 2 },
     },
     assert: {
-        completed: [90000, 200000],
-        cacheHits: [20000, 70000],
+        completed: [300000, 300000],
+        coalescedHit: [8000, 15000],
+        cacheHits: [35000, 45000],
         timeouts: [0, 0],
-        batches: [500, 4000],
-        rss: [90000, 120000],
-        avgBatchSize: [35, 50],
+        batches: [4800, 5300],
+        rss: [50000, 80000],
+        avgBatchSize: [45, 50],
```

Comment:
It maybe worth checking in `sample.txt` so the assertions are reproducible without manually generating a sample.

Reply:
I've included a link to a copy of it: https://github.com/fed135/ha-store/blob/v2.0.1/tests/profiling/README.md

Reply:
There's still a bit of variation based on the user's machine specs, but that should be assertive enough.